pytensor/graph/basic.py

```diff
@@ -936,6 +936,55 @@ def graph_inputs(
     yield from (r for r in ancestors(graphs, blockers) if r.owner is None)
 
 
+def explicit_graph_inputs(
+    graph: Variable[Any, Any] | Iterable[Variable[Any, Any]],
```

Comment:
Do we need the Any, Any? The function above doesn't have it. Tbh I never had timr to check where do these Any come from


```suggestion
    graph: Variable | Iterable[Variable],
```

Reply:
This is what mypy suggested to add as type-hints when the first one that I was trying failed. However, I don't see reason for as to why ```Varaible``` in itself as a type-hint won't work.

Reply:
Let's try it without then